### PR TITLE
Express date ranges as EDTF range in Cocina

### PR DESCRIPTION
**Pull request by jcoyne**
## Why was this change made?

Fixes sul-dlss/happy-heron#541

### CODE REVIEW

File: app/services/cocina/to_fedora/descriptive/event.rb [hunk 1]
```diff
@@ -258,6 +258,8 @@ def write_basic_date(date, cocina_event_type)
         def date_tag(date, cocina_event_type, structured_val_attribs = {})
           value = date.value
           tag = date_type_value_for(date) ? :dateOther : TAG_NAME.fetch(cocina_event_type, :dateOther)
+          encoding = (date.encoding&.code || structured_val_attribs[:encoding]&.code)
```
Comment (mjgiarlo) [inline app/services/cocina/to_fedora/descriptive/event.rb:261]: What is this variable doing? Is it used below?

**Review by mjgiarlo — APPROVED** [review]
have a question about a potentially unused variable, but otherwise good to go

File: app/services/cocina/to_fedora/descriptive/event.rb (hunk 1 above)
Comment (jcoyne) [reply app/services/cocina/to_fedora/descriptive/event.rb:261]: That may be cruft. I used it once, and then refactored.

Comment (jcoyne) [reply app/services/cocina/to_fedora/descriptive/event.rb]: Fixed that.

Comment (mjgiarlo) [reply app/services/cocina/to_fedora/descriptive/event.rb]: :beers:

**Pull request merged**